apps/platform-app/src/app/[locale]/(auth)/(components)/LogOutBtn/LogOutBtn.tsx

```diff
@@ -6,7 +6,11 @@ import { signOut } from 'next-auth/react'
 import { useTranslations } from 'next-intl'
 import { useState } from 'react'
 
-const LogOutBtn = () => {
+interface LogOutBtnProps {
+  onClick?: () => void
+}
+
+const LogOutBtn = ({ onClick }: LogOutBtnProps) => {
   const t = useTranslations(I18nNamespaces.Buttons)
   const [isCalled, setIsCalled] = useState(false)
 
```

Comment:
**issue**: The `onClick` prop should be invoked as a function. Consider using `onClick ? onClick() : signOut({ callbackUrl: AppRoutes.home })` to ensure the function is called correctly.